tools/archives/pyvo_integration/astronomical_archives.xml

```diff
@@ -0,0 +1,1005 @@
+
+            python '$__tool_directory__/astronomical_archives.py' '$output' '$output_csv' '$output_html' '$output_basic_html' '$output_error' $number_of_files '$archiveselection.archive_type' '$archiveselection.archive' '$query_section.query_selection.query_type' '$query_section.query_selection.dataproduct_type' '$query_section.query_selection.obs_collection' '$query_section.query_selection.facility_name' '$query_section.query_selection.instrument_name' '$query_section.query_selection.em_min' '$query_section.query_selection.em_max' '$query_section.query_selection.target_name' '$query_section.query_selection.obs_publisher_id' '$query_section.query_selection.s_fov' '$query_section.query_selection.calibration_level' '$query_section.query_selection.order_by' '$query_section.query_selection.obs_title' '$query_section.query_selection.obs_id' '$query_section.query_selection.t_min' '$query_section.query_selection.t_max' '$query_section.query_selection.dataproduct_type_select' '$output_csv' '$output_html'
+
+          #end if
+
+        #elif $query_section.query_selection.query_type == 'raw_query':
+
+          #if $archiveselection.archive_type == 'registry' :
+
+            python '$__tool_directory__/astronomical_archives.py' '$output' '$output_csv' '$output_html' '$output_basic_html' '$output_error' $number_of_files '$archiveselection.archive_type' '$archiveselection.keyword' '$archiveselection.wavebands' '$archiveselection.service_type' '$query_section.query_selection.query_type' '$query_section.query_selection.table' '$query_section.query_selection.where_clause.where_field' '$query_section.query_selection.where_clause.where_condition' '$query_section.query_selection.url_field' '$output_csv' '$output_html'
+
+          #elif $archiveselection.archive_type == 'archive' :
+
+            python '$__tool_directory__/astronomical_archives.py' '$output' '$output_csv' '$output_html' '$output_basic_html' '$output_error' $number_of_files '$archiveselection.archive_type' '$archiveselection.archive' '$query_section.query_selection.query_type' '$query_section.query_selection.table' '$query_section.query_selection.where_clause.where_field' '$query_section.query_selection.where_clause.where_condition' '$query_section.query_selection.url_field' '$output_csv' '$output_html'
+
+          #end if
+
+        #end if
+
+        python '$__tool_directory__/astronomical_archives.py' '$output' '$output_csv' '$output_html' '$output_basic_html' '$output_error'
+      ]]>
+    </command>
+    <inputs>
+        <conditional name="archiveselection">
+            <param name="archive_type" type="select" label="Archive Selection">
+              <option value="registry">Query IVOA registry</option>
+              <option value="archive">Query specific IVOA archive</option>
+            </param>
+            <when value="registry">
+              <param name="keyword" type="text" label="Keyword" />
+              <param name="service_type" type="select" label="Service type">
```

Comment:
please put a reference for versionned service type vocabulary. also for all other select options which rely on a particular version of some vocabularies